ouroboros-consensus-shelley/test/Test/ThreadNet/RealTPraos.hs

```diff
@@ -172,13 +235,21 @@ mkGenesisConfig k maxKESEvolutions coreNodes = ShelleyGenesis {
     , sgSlotLength            = tpraosSlotLength
     , sgUpdateQuorum          = 1  -- TODO
     , sgMaxMajorPV            = 1000 -- TODO
-    , sgMaxLovelaceSupply     = 1000 -- TODO
-    , sgMaxBodySize           = 1000 -- TODO
+    , sgMaxLovelaceSupply     = maxLovelaceSupply
+    , sgMaxBodySize           = 10000 -- TODO
     , sgMaxHeaderSize         = 1000 -- TODO
     , sgGenDelegs             = coreNodesToGenesisMapping
-    , sgInitialFunds          = Map.empty -- TODO
+    , sgInitialFunds          = initialFunds
     }
   where
+    initialLovelacePerCoreNode :: Word64
+    initialLovelacePerCoreNode = 1000
+
+     -- TODO
```

Comment:
Is this still TODO? I remember reading somewhere that it should be higher than the initial funds because it needs to include rewards.

Reply:
It's still TODO to think properly about it and what it should be set to. I assume you mean reserves rather than rewards?

Reply:
> It's still TODO to think properly about it and what it should be set to.

Ok, good.

> I assume you mean reserves rather than rewards?

Ah, I remember where I read it: https://github.com/input-output-hk/cardano-node/pull/798/files#diff-d6f138a206ea4a3c592b3faf15a1dc13R60

Reply:
Ah, right. That makes sense, because rewards are paid from the reserves.